modules/nw-ovn-kubernetes-get-raft-db-status.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-architecture.adoc
+
+:_content-type: PROCEDURE
+[id="nw-list-querying-status-northbound-southbound-databases_{context}"]
+= Querying the status of the northbound and southbound databases
+
+The Kubernetes control plane nodes each contain an `ovnkube-master` pod which hosts containers for the OVN northbound and southbound databases which must form `Raft` consensus between its peer database. For example all OVN northbound databases must form a `Raft` cluster and all southbound databases must form a separate `Raft` cluster. The OVN northbound and southbound databases must have distinct `Raft` leaders.
+
+.Prerequisites
+
+* Access to the cluster as a user with the cluster-admin role.
+* The OpenShift CLI (oc) installed.
+
+.Procedure
+
+. Run the following command to get all resources in the OVN-Kubernetes project:
++
+[source,terminal]
+----
+$ oc get all -n openshift-ovn-kubernetes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                       READY   STATUS    RESTARTS   AGE
+pod/ovnkube-master-5mxg7   6/6     Running   0          133m
+pod/ovnkube-master-mtjjz   6/6     Running   0          133m
+pod/ovnkube-master-qhbfk   6/6     Running   0          133m
+pod/ovnkube-node-9rwkr     5/5     Running   0          133m
+pod/ovnkube-node-hrq8n     5/5     Running   0          123m
+pod/ovnkube-node-jttc7     5/5     Running   0          123m
+pod/ovnkube-node-km8bl     5/5     Running   0          123m
+pod/ovnkube-node-mb752     5/5     Running   0          133m
+pod/ovnkube-node-sxcx8     5/5     Running   0          133m
+
+NAME                            TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)             AGE
+service/ovn-kubernetes-master   ClusterIP   None         <none>        9102/TCP            133m
+service/ovn-kubernetes-node     ClusterIP   None         <none>        9103/TCP,9105/TCP   133m
+service/ovnkube-db              ClusterIP   None         <none>        9641/TCP,9642/TCP   133m
+
+NAME                            DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR                                                 AGE
+daemonset.apps/ovnkube-master   3         3         3       3            3           beta.kubernetes.io/os=linux,node-role.kubernetes.io/master=   133m
+daemonset.apps/ovnkube-node     6         6         6       6            6           beta.kubernetes.io/os=linux                                   133m
+----
+
+. Select one of the `ovnkube-master` pods and run the following to determine the status of the OVN northbound database:
```

Comment:
We're really getting the role on a node.  One of them must be the 'leader.'  If not then we have an issue.  You might reference that there is a readinessProbe for this and link to that section.

Reply:
updated